### DISCUSSION TITLE
part-ii/2018-112/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#44] [Question comments]
Part II 2018 Principles of Quantum Mechanics: Paper 4, Section II, D 

https://questions.tripos.org/part-ii/2018-112/

**Comment by silkca** [comment]
For the first part, it suffices to show that $S_{\pm}|s, \sigma\rangle$ is indeed an eigenstate of $\mathbf{S}^{2}$ and $S_{z}$ with the relevant eigenvalues, and then calculate $||S_{\pm}|s, \sigma\rangle||^2$. Then, it is simple to find the action of $S_{x}$ on each $|1,\sigma\rangle$ by writing $S_{x}$ in terms of $S_+$ and $S_-$.

The last part I found difficult, but I have been informed since that it isn't particularly: it essentially consists of some linear algebra performed on $\mathbf{n}\cdot\mathbf{S}$.

**Reply by thenewtonium** [reply]
To elaborate on the last part: if we write the eigenstate of $\mathbf{n}\cdot\mathbf{S}$ that we're trying to find as $a|1,-1\rangle + b|1,0\rangle + c|1,1\rangle$, then the linear system we get has an equation involving $a$ and $b$ only, as well as an equation involving $c$ and $b$ only. The algebra is a lot nicer if you equate these two equations, rather than using the other one involving $a$, $b$, and $c$.... (as I did on my attempt oops).
